### PR TITLE
Юнин - Лабораторная работа #3

### PR DESCRIPTION
**Pull request by IvanYunin**

### CODE REVIEW

**Comment by IvanYunin** [comment]
@Enreyz @Pahandrovich @Agricola87 
Review please.

**Comment by Enreyz** [comment]
Good code :+1:

**Comment by lesya89** [comment]
👍

**Comment by Pahandrovich** [comment]
👍

File: modules/temperature-converter/src/temperature_calculator.cpp [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    message_ =
+        std::string(message) +
+          "This is a temperature calculator application.\n" +
+          "Please provide arguments in the following format:\n"+
+
+          "  $ " + appname + " <value> <operation>\n" +
+
+          "Value have to be a double-precision number, " +
+          "and <operation> is one of the following:\n" +
+          "'CelsiusToFahrenheit',\n" +
```
Comment (Agricola87) [inline modules/temperature-converter/src/temperature_calculator.cpp]: Is it necessary to alternate different type of quotations?

File: modules/temperature-converter/src/temperature_calculator.cpp [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+}
+
+std::string TemperatureCalculator::operator()(int argc, const char** argv) {
+    Arguments args;
+
+    if (!ValidateNumberOfArgs(argc, argv)) {
+        return message_;
+    }
+    try {
+        args.value      = parseDouble(argv[1]);
```
Comment (Agricola87) [inline modules/temperature-converter/src/temperature_calculator.cpp]: format: Redundant spaces

File: modules/temperature-converter/src/temperature_calculator.cpp (hunk 1 above)
Comment (IvanYunin) [reply modules/temperature-converter/src/temperature_calculator.cpp]: Don't understand the question

File: modules/temperature-converter/src/temperature_calculator.cpp (hunk 2 above)
Comment (IvanYunin) [reply modules/temperature-converter/src/temperature_calculator.cpp]: Done!

File: modules/temperature-converter/src/temperature_calculator.cpp (hunk 1 above)
Comment (Agricola87) [reply modules/temperature-converter/src/temperature_calculator.cpp]: It's ok.

File: modules/temperature-converter/test/test_temperature_calculator.cpp [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+    }
+
+ private:
+    TemperatureCalculator app_;
+    string output_;
+};
+
+TEST_F(TemperatureCalculatorTest, Can_Print_Help_Without_Arguments) {
+    vector<string> args = {};
+
```
Comment (Agricola87) [inline modules/temperature-converter/test/test_temperature_calculator.cpp]: Add comments, like below

Comment (IvanYunin) [reply modules/temperature-converter/test/test_temperature_calculator.cpp]: Done!

**Pull request merged**